src/routes/open/container/Open.tsx

```diff
@@ -92,6 +93,9 @@ export const createSafe = (values: CreateSafeValues, userAccount: string): Promi
   promiEvent
     .once('transactionHash', (txHash) => {
       saveToStorage(SAFE_PENDING_CREATION_STORAGE_KEY, { txHash, ...values })
+      txMonitor({ sender: userAccount, hash: txHash, data: deploymentTx.encodeABI() }, (txReceipt) => {
+        console.log({ txReceipt })
```

Comment:
When there's a speed up tx, this callback is called but we don't have a way to pass the txReceipt to the .then()

Reply:
I think we should convert this `promiEvent` to a regular promise/async call.
Web3's PromiEvent is an extremely weird creation.